src/frontend/web_application/src/components/form/DatePickerGroup/presenter.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import DatePicker from 'react-datepicker'; // https://github.com/Hacker0x01/react-datepicker
+import 'react-datepicker/dist/react-datepicker.css';
+import moment from 'moment';
+import classnames from 'classnames';
+
+import './style.scss';
+
+class DatePickerGroup extends Component {
+  static propTypes = {
+    calendarClassName: PropTypes.string,
+    className: PropTypes.string,
+    id: PropTypes.string.isRequired,
+    inputClassName: PropTypes.string,
+    label: PropTypes.string,
+    locale: PropTypes.string,
+    onDateChange: PropTypes.func.isRequired,
+    selected: PropTypes.instanceOf(Date),
+  };
+  static defaultProps = {
+    calendarClassName: null,
+    className: null,
+    inputClassName: null,
+    label: null,
+    locale: undefined,
+    selected: null,
+  };
```

Comment:
Is this a common practice you have in Caliopen? Seems unnecessary to me. But one could like it for some reasons :)